### ISSUE TITLE
How to get the RGB images?

### ISSUE THREAD

**Issue by BingHan0458** [xcyan/nips16_PTN#14]
Very great work!
I want to ask, is your RGB images rendered by yourself? Is there color information?
If I want to get a dataset which includes RGB images and point clouds of each single object, is there such a data set?
Looking forward to your reply!
Thank you!

**Comment by anusha-devulapally** [comment]
Hi @BingHan0458, I am currently looking for a similar dataset (RGB and point cloud for each single object).  Have you come across any or worked with them? If so, please do point me to it. Thank you and looking forward for your reply

**Comment by BingHan0458** [comment]
Yes, I got some point cloud datasets with RGB and coordinate, including a synthesis dataset [DensePoint](https://github.com/Yochengliu/DensePoint) and a real-scanned dataset [OmniObject3D](https://opendatalab.com/OpenXDLab/OmniObject3D-New). I don't know what point cloud tasks you want to do, but I hope it will be helpful to you.
